src/main/java/org/apache/xmlbeans/impl/schema/StscComplexTypeResolver.java

```diff
@@ -1002,7 +1002,7 @@ static void translateAttributeModel(XmlObject parseTree,
     static SchemaParticle extendContentModel(SchemaParticle baseContentModel, SchemaParticle extendedContentModel, XmlObject parseTree) {
         // http://www.w3.org/TR/xmlschema-1/#element-complexContent::extension
 
-        // 2.1 If the explicit content is empty, then the {content type} of the type definition resolved to by the �actual value� of the base [attribute]
```

Comment:
* please move all the unrelated comment changes to a separate PR
* see also [StscChecker.java](https://github.com/apache/xmlbeans/pull/14/files#diff-ae89ce4a13e3f4598d0194de6d0e952c8113d8c83be018e915aefb5e038da16a)

Reply:
Sorry, I was in a hurry to submit the PR and didn't review it carefully. I am fixing these issues and will make a new commit soon.